### PR TITLE
[ADD] stock_forecast_report: Backport from 14.0

### PR DESCRIPTION
**Pull request by chienandalu**
This is a backport from the 14.0 Odoo core forecast report adapted to the limitations of this version. It substitutes the useless current forecast report.

image

cc @Tecnativa TT28367

### CODE REVIEW

**Comment by victoralmau** [comment]
Travis is red, need to create `README.rst` file.

File: stock_forecast_report/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Odoo S.A.
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Forecast Report",
+    "summary": "Backport of core 14.0 Forecast Report",
+    "version": "12.0.1.0.0",
+    "category": "Inventory",
+    "website": "https://github.com/OCA/product-attribute",
```
Comment (victoralmau) [inline stock_forecast_report/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/stock-logistics-reporting",
```

File: stock_forecast_report/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+To use this module, you need to:
+
+#. Go to *Inventory > Reports > Stock Forecast*
```
Comment (victoralmau) [inline stock_forecast_report/readme/USAGE.rst]:
```suggestion
#. Go to *Inventory > Reporting > Forecasted Inventory*
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Review by victoralmau — APPROVED** [review]

**Comment by victoralmau** [comment]
Travis red now, need to fix it.

**Comment by chienandalu** [comment]
Travis is happy now :smile:

**Review by marcelsavegnago — APPROVED** [review]

**Comment by chienandalu** [comment]
Don't merge yet, please

**Review by ernestotejeda — APPROVED** [review]

**Comment by marcelsavegnago** [comment]
> Don't merge yet, please

Wouldn't it be better to change PR to draft?

**Comment by chienandalu** [comment]
Ok, it's ready now. I needed to make a default filter to avoid rendering to much information. When we put it to test in a customer db with thousands of products an stock moves, the browse just couldn't render properly the graph. I wonder if it will happen in v14 as well but I guess it will.

File: stock_forecast_report/readme/ROADMAP.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+* This is a backport from the 14.0 Odoo core forecast report.
```
Comment (pedrobaeza) [inline stock_forecast_report/readme/ROADMAP.rst]:
```suggestion
* This won't be needed on 14.0+.
```

**Review by pedrobaeza — COMMENTED** [review]
Please add usage instructions.

File: stock_forecast_report/readme/ROADMAP.rst (hunk 3 above)
Comment (chienandalu) [reply stock_forecast_report/readme/ROADMAP.rst]: done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**